pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/JuniperSetupUtil.java

```
@@ -0,0 +1,325 @@
+                    stmt.setString(2, studyGuid);
+                    stmt.setString(3, displayName);
+                    stmt.setString(4, collaboratorPrefix);
+                    stmt.executeUpdate();
+                    try (ResultSet rs = stmt.getGeneratedKeys()) {
+                        if (rs.next()) {
+                            ddpInstanceId = rs.getString(1);
+                        }
+
+                    } catch (Exception e) {
+                        throw new RuntimeException(e);
+                    }
+                } catch (SQLException e) {
+                    throw new RuntimeException("Error inserting ddp_instance for juniper group in ddp_instance ", e);
+                }
+            } catch (SQLException ex) {
+                throw new RuntimeException("Error inserting ddp_instance for juniper group in ddp_instance ", ex);
+            }
+            if (ddpGroupId == null || ddpInstanceId == null) {
+                throw new RuntimeException("Something went wrong");
+            }
+            try (PreparedStatement stmt = conn.prepareStatement(INSERT_DDP_INSTANCE_GROUP, Statement.RETURN_GENERATED_KEYS)) {
+                try {
+                    stmt.setString(1, ddpInstanceId);
+                    stmt.setString(2, ddpGroupId);
+                    stmt.setString(3, ddpGroupId);
+                    stmt.executeUpdate();
+                    try (ResultSet rs = stmt.getGeneratedKeys()) {
+                        if (rs.next()) {
+                            ddpInstanceGroupId = rs.getString(1);
+                        }
+
+                    } catch (Exception e) {
+                        throw new RuntimeException(e);
+                    }
+                } catch (SQLException e) {
+                    throw new RuntimeException("Error inserting in ddp_instance_group ", e);
+                }
+            } catch (SQLException ex) {
+                dbVals.resultException = ex;
```

This is not useful. This error will just be lost.
Assuming this code will only be used to setup tests, I suggest not catching these exceptions -- just let them bubble up to the test methods. Or you could create a method that sits on top of this and consolidates the exceptions into a DsmInternalError.